pom.xml

```diff
@@ -65,1442 +65,1442 @@
-  </dependency>
-  <dependency>
-    <groupId>org.igniterealtime.smack</groupId>
-    <artifactId>smack-extensions</artifactId>
-    <version>4.1.6</version>
-    <scope>provided</scope>
-  </dependency>
-<!-- Xmpp end -->
-
-  </dependencies>
+<dependencies>
+
+<!-- AudioFile begin -->
+  <dependency>
+    <groupId>javazoom</groupId>
+    <artifactId>jlayer</artifactId>
+    <version>1.0.1</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>com.googlecode.soundlibs</groupId>
+    <artifactId>mp3spi</artifactId>
+    <version>1.9.5.4</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>com.googlecode.soundlibs</groupId>
+    <artifactId>vorbisspi</artifactId>
+    <version>1.0.3.3</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- AudioFile end -->
+
+<!-- AzureTranslator begin -->
+  <dependency>
+    <groupId>io.github.firemaples</groupId>
+    <artifactId>microsoft-translator-java-api</artifactId>
+    <version>0.8.3</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- AzureTranslator end -->
+
+<!-- BodyPart begin -->
+  <dependency>
+    <groupId>org.apache.commons</groupId>
+    <artifactId>commons-lang3</artifactId>
+    <version>3.3.2</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- BodyPart end -->
+
+<!-- BoofCv begin -->
+  <dependency>
+    <groupId>org.boofcv</groupId>
+    <artifactId>boofcv-core</artifactId>
+    <version>0.31</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>org.boofcv</groupId>
+    <artifactId>boofcv-swing</artifactId>
+    <version>0.31</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>org.boofcv</groupId>
+    <artifactId>boofcv-openkinect</artifactId>
+    <version>0.31</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- BoofCv end -->
+
+<!-- ChessGame begin -->
+  <dependency>
+    <groupId>ChessBoard</groupId>
+    <artifactId>ChessBoard</artifactId>
+    <version>1.0.0</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- ChessGame end -->
+
+<!-- CleverBot begin -->
+  <dependency>
+    <groupId>ca.pjer</groupId>
+    <artifactId>chatter-bot-api</artifactId>
+    <version>2.0.1</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>com.squareup.okhttp3</groupId>
+    <artifactId>okhttp</artifactId>
+    <version>3.9.0</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- CleverBot end -->
+
+<!-- Cron begin -->
+  <dependency>
+    <groupId>it.sauronsoftware.cron4j</groupId>
+    <artifactId>cron4j</artifactId>
+    <version>2.2.5</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- Cron end -->
+
+<!-- CsvConnector begin -->
+  <dependency>
+    <groupId>net.sf.opencsv</groupId>
+    <artifactId>opencsv</artifactId>
+    <version>2.3</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- CsvConnector end -->
+
+<!-- Deeplearning4j begin -->
+  <dependency>
```

Comment:
Do you know why the whitespace difference, was this generated

Reply:
Was the file generated 100% before?  This file is the result of running MavenWrapper.xml  no changes.